Makefile.common.mk

```diff
@@ -4,5 +4,5 @@
 #
 #----------------------------------------------------------------------------
 
-EXTENSION        	:= pg_sphere
-PGSPHERE_VERSION 	:= 1.5.1
+EXTENSION			:= pg_sphere
```

Comment:
This whitespace change doesn't look right on GitHub, but I take it looks right in your editor?

Reply:
@esabol I just replaced spaces with tabs. It was ok for me because my editor used 4 spaces for tab. I will roll back this change. Thank you.